snuba/migrations/operations.py

```diff
@@ -17,8 +18,21 @@ def __init__(self, storage_set: StorageSetKey):
     def storage_set(self) -> StorageSetKey:
         return self._storage_set
 
-    def execute(self, local: bool) -> None:
-        cluster = get_cluster(self._storage_set)
+    def execute(self, local: bool, partition_id: Optional[int]) -> None:
```

Comment:
I think we should ensure to be very verbose in the log reminding to the user what partition they are running migrations on. 
Example:
- when listing the migrations, please ensure the output mentions the partition
- when doing a dry-run, same
- when executing a migraiton, please mention in the output which partition the migration was run.